src/status_im/contexts/wallet/networks/events.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.wallet.networks.events
+  (:require [re-frame.core :as rf]
+            [status-im.contexts.wallet.networks.db :as networks.db]
+            [status-im.contexts.wallet.rpc-data-store.networks :as rpc-data-store.networks]))
+
+(rf/reg-event-fx
+ :wallet/get-ethereum-chains
+ (fn [_]
+   {:json-rpc/call
+    [{:method     "wallet_getEthereumChains"
```

Comment:
FYI: We need to update this RPC. The desktop has been using `getFlatEthereumChains` RPC.

Reply:
I started working on the chain management issue, and there I replaced it to the new RPC, since I had to restructure the networks db structure a bit :+1: